apisix/plugins/ocsp-stapling.lua

```diff
@@ -199,6 +265,19 @@ function _M.init()
                 type = "boolean",
                 default = false,
             },
+            ssl_stapling_responder = {
+                type = "string",
+                pattern = [[^http://]],
+            },
+            ssl_ocsp = {
+                type = "string",
+                default = "off",
+                enum = {"off", "leaf"},
+            },
+            ssl_ocsp_responder = {
```

Comment:
these fields are just for customising the values of default nginx directives right?

Reply:
yes, but no all same as nginx directives, just part of it, nginx directives allows "off", "leaf" and "on". the openresty api can not get full chain cert, so we don't support "on" here.